files/validate_settings.rb

```diff
@@ -37,6 +37,7 @@
 
 # Validate the ServiceNow credentials
 begin
+  File.write('/tmp/settings', settings)
```

Comment:
This feels like it's probably a testing remnant?

Reply:
Yup, it is. Thanks!